src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+
+  <div className="CommentList">
+
+    {comments.length
+      ? comments.map(comment => (
```

Comment:
I think it's better not to render CommentList component at all if there are no comments. 
Would be great to move this check to the parent component